packages/components-css/heading-1-css/src/_mixins.scss

```diff
@@ -0,0 +1,3 @@
+@mixin nl-heading-1 {
+  font-size: var(--nl-heading-1-font-size, revert);
+}
```

Comment:
waarom `_mixins.scss` en niet `_mixin.scss` zoals nu in de design-tokens packages wordt gebruikt en bij Utrecht?

Reply:
Fixed.